test/integration/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "integration",
+  "version": "1.0.0",
+  "description": "Integration tests for the Rev.ai node sdk",
+  "main": "index.js",
+  "jest": {
+    "testEnvironment": "node"
+  },
+  "scripts": {
+    "test": "jest"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/revdotcom/revai-node-sdk.git"
+  },
+  "author": "Rev Ai",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/revdotcom/revai-node-sdk/issues"
+  },
+  "homepage": "https://github.com/revdotcom/revai-node-sdk#readme",
+  "devDependencies": {
+    "revai-node-sdk": "^2.0.1",
```

Comment:
In order to run theses on the latest version of the SDK we will probably have to import the SDK from the dist folder.